x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterEnrichIT.java

```diff
@@ -402,6 +402,33 @@ public void testTopNThenEnrichRemote() {
             assertThat(executionInfo.clusterAliases(), equalTo(Set.of("", "c1", "c2")));
             assertCCSExecutionInfoDetails(executionInfo);
         }
+
+        // No renames, no KEEP
```

Comment:
Might be good to make a new test out of this, possibly adding a reference to why we add this test (optional). And/or detail why KEEP is "detrimental" to the test. 🙏